package.json

```
@@ -44,58 +47,60 @@
     "electron-forge": "^5.1.1",
     "electron-prebuilt-compile": "2.0.2",
     "electron-rebuild": "^1.5.7",
-    "ember-ajax": "^3.0.0",
+    "ember-ajax": "^3.1.0",
     "ember-browserify": "^1.1.12",
-    "ember-cli": "2.17.1",
+    "ember-cli": "~3.5.0",
```

Can you pin this to `3.5.0` instead of `~3.5.0`?
```suggestion
    "ember-cli": "3.5.0",
```